bin/anvi-run-ncbi-cogs

```diff
@@ -19,9 +19,10 @@ __copyright__ = "Copyleft 2015-2018, the Meren Lab (http://merenlab.org/)"
 __license__ = "GPL 3.0"
 __version__ = anvio.__version__
 __authors__ = ['meren']
-__requires__ = ['contigs-db', 'cogs-data',]
+__requires__ = ['cogs-data',]
 __provides__ = ['functions',]
```

Comment:
Provides should now include `functions-txt`, as well :)

Reply:
Done.